pkg/hubble/filters/http.go

```diff
@@ -101,6 +102,39 @@ func filterByHTTPMethods(methods []string) (FilterFunc, error) {
 	}, nil
 }
 
+func filterByHTTPPaths(pathRegexpStrs []string) (FilterFunc, error) {
+	pathRegexps := make([]*regexp.Regexp, 0, len(pathRegexpStrs))
+	for _, pathRegexpStr := range pathRegexpStrs {
+		pathRegexp, err := regexp.Compile(pathRegexpStr)
+		if err != nil {
+			return nil, fmt.Errorf("%s: %v", pathRegexpStr, err)
+		}
+		pathRegexps = append(pathRegexps, pathRegexp)
+	}
+
+	return func(ev *v1.Event) bool {
+		http := ev.GetFlow().GetL7().GetHttp()
+
+		if http == nil || http.Url == "" {
+			return false
+		}
+
+		uri, err := url.ParseRequestURI(http.Url)
```

Comment:
Arguably, instead of matching on just the path, we could match on the whole URI instead, which would allow the user to match on query parameters. One downside to this would be that the regular expressions might match query parameters that contain path-like strings. For example, it requires extra care to write a regexp that matches `/post/` but not `/page/?name=/post/`.

Reply:
Your implementation seems sane to me. We can add other filters to deal with query strings and such